README.md

```diff
@@ -24,8 +24,12 @@ uvicorn main:app --reload
 ## **Active Endpoints**
 1. ```POST /menu```
     - Creates menu object and stores in database instance.
-2. ```More endpoints coming soon```
-
+2. ```POST /rest_database```
```

Comment:
Should be reset_database

Reply:
good catch let me fix and create a new PR